xbmc/Application.cpp

```diff
@@ -1569,7 +1569,8 @@ void CApplication::Render()
     infoMgr.GetInfoProviders().GetSystemInfoProvider().UpdateFPS();
   }
 
-  CServiceBroker::GetWinSystem()->GetGfxContext().Flip(hasRendered, m_appPlayer.IsRenderingVideoLayer());
+  if (!m_skipGuiRender)
```

Comment:
m_skipGuiRender is a hack for raspberry pi and won't ever be true on other platforms. Even if is was, this change is wrong because m_skipGuiRender has direct impact on hasRendered.

Reply:
Its used for android, too

Reply:
the only place I can find where it is set to true is: https://github.com/xbmc/xbmc/blob/cad2a7d4363dbf65b1013572d17c648749b17a19/xbmc/Application.cpp#L2397